.eslintrc

```diff
@@ -1,73 +1,16 @@
 {
-  "extends": ["eslint:all", "plugin:import/recommended"],
+  "extends": [
+    "eslint:recommended"
```

Comment:
You removed the `import` plugin from the config but it's still installed as a `devDependency`. Could we keep using it, I think it is pretty helpful?

Reply:
Will add it back, funny that I had copied over this whole setup from another repo we have, need to check which.

Reply:
I know we are not consistent on this :-(

Reply:
Updated

Reply:
I think it's still missing in the `extends` array :thinking:

Reply:
Of course it is 🤦‍♀️ Added, thanks

Reply:
:+1: